components/formats-api/src/loci/formats/in/DynamicMetadataOptions.java

```diff
@@ -476,5 +491,61 @@ public File getFile(String name, File defaultValue) {
     }
     return new File(val);
   }
+  
+  public void loadOptions(File optionsFile) throws IOException, FormatException {
+    if (!optionsFile.exists()) {
+      LOGGER.trace("Options file doesn't exist: {}", optionsFile);
+      // TODO: potentially create option
+      return;
+    }
+    if(!optionsFile.canRead()) {
+      LOGGER.trace("Can't read options file: {}", optionsFile);
+      return;
+    }
+
+    // locate an input stream
+    InputStream stream = null;
+    try {
+      stream = new FileInputStream(optionsFile);
+    } catch (FileNotFoundException e) {
+      LOGGER.debug(e.getMessage());
+      return;
+    }
+
+    // Read options from file
+    BufferedReader in = null;
+    in = new BufferedReader(new InputStreamReader(stream, Constants.ENCODING));
+    while (true) {
+      String line = null;
+      line = in.readLine();
+      if (line == null) break;
+      // Ignore characters following # sign (comments)
+      int ndx = line.indexOf('#');
+      if (ndx >= 0) line = line.substring(0, ndx);
+      line = line.trim();
+      if (line.equals("")) break;
+
+      StringTokenizer st1 = new StringTokenizer(line, "=");
+      while (st1.hasMoreTokens()) {
+        String key = st1.nextToken();
+        if (st1.hasMoreTokens()) {
+          set(key, st1.nextToken());
+        }
+      }
+    }
```

Comment:
Worth using ```IniParser``` here?

Reply:
Good call, no point reinventing the wheel, makes life much simpler